.github/CODEOWNERS

```diff
@@ -11,3 +11,5 @@ dags/solutions_team/configs/tensorflow @chandrasekhard2 @EricLeFort
 dags/pytorch_xla @will-cromar
 
 dags/multipod @jonb377
+
+dags/mlcompass @ortibazar @sganeshb @brajiang @wlzhg
```

Comment:
It shows the error. Could you help ensure they have accepted my invite as contributors in this repo? Thank you!